src/main/java/org/mongounit/MongoUnitUtil.java

```diff
@@ -176,12 +176,11 @@ public static void toDatabase(
   public static void dropAllCollectionsInDatabase(MongoDatabase mongoDatabase) {
 
     // Iterate over all collections in the db and drop them
-    for (String collectionName : mongoDatabase.listCollectionNames()) {
-
+    mongoDatabase.listCollectionNames().forEach(collectionName -> {
```

Comment:
@alexisGandar I am assuming this is just a newer syntax cleanup? This really doesn't have anything to do with the version update, does it?